server/app/assets/javascripts/preview.ts

```diff
@@ -34,6 +34,8 @@ class PreviewController {
   private static readonly DEFAULT_ENTITY_TYPE = 'Sample repeated entity type'
   private static readonly DEFAULT_OPTION_TEXT = 'Sample question option'
 
+  private static readonly STATIC_QUESTION_TYPE = 'Static Text'
```

Comment:
**suggestion (non-blocking):** Related can you comment above the previous three on L33 that they come from `{@link ApplicantQuestionRendererFactory}`

Reply:
Done